frontend/src/components/common/QuestionCard.tsx

```diff
@@ -78,7 +78,7 @@ const QuestionCard = ({
           </Flex>
           <List
             spacing={3}
-            styleType="lower-alpha"
+            styleType={questions.length > 1 ? "lower-alpha" : "none"}
```

Comment:
questions with only one sub-question should not be numbered. (i.e should not have `a.` before the question if there is only one question)